### PR TITLE
+doc update paradox and enable source url feature

### PR DESCRIPTION
**Pull request by ktoso**

### CODE REVIEW

**Comment by ktoso** [comment]
Thanks a lot @dwijnand @eed3si9n!

Preview:

java_documentation_ _akka_http

**Comment by ktoso** [comment]
FYI @jonas, you may like this :-)

**Comment by jrudolph** [comment]
Wow nice, I though about building it myself a while ago. This way is nicer ;)

**Comment by jonas** [comment]
I do like this ... and props to @gsechaud for adding this to Paradox.

File: docs/src/main/paradox/_template/page.st [hunk 1]
```diff
@@ -72,7 +72,11 @@
             <div class="page-content row">
               <div class="small-12 large-9 column" id="docs">
                 $page.content$
-                <!-- FIXME copy the template again once paradox 0.2.8 is out, it has source inclusion here.-->
+                
+                <div>
+                  $source()$
```
Comment (ktoso) [inline docs/src/main/paradox/_template/page.st]: We can customise the text too but going with the simple thing now.

**Comment by ktoso** [comment]
Merging, full steam ahead ;-)

**Pull request merged**